packages/runtime/src/helpers/files.ts

```diff
@@ -467,3 +478,10 @@ export const movePublicFiles = async ({
     await copy(publicDir, `${publish}${basePath}/`)
   }
 }
+
+export const removeMetadataFiles = async (publish: string) => {
```

Comment:
Would it not be faster to `await Promise.all()` or `await Promise.allSettled()`?

Reply:
Generally yes, but if we don't want to do 1 by 1 for perf reasons we should still limit concurrency as otherwise some starvation might happen and doing `Promise.all()` might actually be slower than doing it 1 by 1

For files moving we do have concurency of number of cpu cores, with minimum of 2 (if on single core):

https://github.com/netlify/next-runtime/blob/f051e227ca10ddc228b1d044eb835fe66fd1d733/packages/runtime/src/helpers/files.ts#L152-L153

So I would suggest we do the same here?

Reply:
Swapped to `p-limit` (like snippet above) in https://github.com/netlify/next-runtime/pull/2104/commits/d709eb1dde0e2fccd2304b65de19eb1428ef17db